### PR TITLE
fix: build with Xcode 12.4 SQPIT-781

### PR DESCRIPTION
**Pull request by billypchan**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/SQPIT-781" title="SQPIT-781" target="_blank">TaskSQPIT-781</a>  Build frameworks compiles on both Xcode 12 and 13
  </td></table>
  <br />
 

# What's new in this PR?

ref: https://github.com/wireapp/wire-ios-protos/pull/60

### CODE REVIEW

File: Tests/Source/Notifications/PushNotifications/ZMLocalNotificationTests_Message.swift [hunk 1]
```diff
@@ -707,11 +707,11 @@ extension ZMLocalNotificationTests_Message {
...
-        #if targetEnvironment(simulator)
-        if #available(iOS 15, *) {
-            XCTExpectFailure("Expect to fail on iOS 15 simulator. ref: https://wearezeta.atlassian.net/browse/SQCORE-1188")
-        }
-        #endif
+//        #if targetEnvironment(simulator)
+//        if #available(iOS 15, *) {
+//            XCTExpectFailure("Expect to fail on iOS 15 simulator. ref: https://wearezeta.atlassian.net/browse/SQCORE-1188")
+//        }
+//        #endif
```
Comment (billypchan) [inline Tests/Source/Notifications/PushNotifications/ZMLocalNotificationTests_Message.swift:714]:
```suggestion
#if targetEnvironment(simulator)
        if #available(iOS 15, *) && swift(>=5.4) {
            XCTExpectFailure("Expect to fail on iOS 15 simulator. ref: https://wearezeta.atlassian.net/browse/SQCORE-1188")
        }
#endif
```

File: Tests/Source/Notifications/PushNotifications/ZMLocalNotificationTests_Message.swift [hunk 2]
```diff
@@ -706,12 +706,11 @@ extension ZMLocalNotificationTests_Message {
     }
 
     func testThatItGeneratesCorrectCategoryIfEncryptionAtRestIsEnabledForTeamUser() throws {
-        // GIVEN
-        #if targetEnvironment(simulator)
+#if targetEnvironment(simulator) && swift(>=5.4)
```
Comment (billypchan) [inline Tests/Source/Notifications/PushNotifications/ZMLocalNotificationTests_Message.swift:709]:
```suggestion
// GIVEN
#if targetEnvironment(simulator) && swift(>=5.4)
```

**Review by agisilaos — APPROVED** [review]

**Comment by billypchan** [comment]
release patch

**Comment by zenkins** [comment]
Hello, @billypchan! Merging PR for command **release patch** for PR #**1627**. Will create a **patch** release after merged.
Next version is 406.1.2.

**Comment by billypchan** [comment]
release patch

**Comment by zenkins** [comment]
Hello, @billypchan! Merging PR for command **release patch** for PR #**1627**. Will create a **patch** release after merged.
Next version is 406.1.2.

**Pull request merged**